src/vs/workbench/services/workingCopy/common/workingCopyService.ts

```diff
@@ -139,8 +139,10 @@ export class WorkingCopyService extends Disposable implements IWorkingCopyServic
 	registerWorkingCopy(workingCopy: IWorkingCopy): IDisposable {
 		const disposables = new DisposableStore();
 
-		// {{SQL CARBON EDIT}} @chlafreniere need to block working copies of notebook editors from being tracked
-		if (workingCopy.resource.path.includes('notebook-editor-') && workingCopy.resource.scheme === Schemas.untitled) {
+		// {{SQL CARBON EDIT}} @chlafreniere need to block working copies of notebook editors from being tracked.
+		// {{SQL CARBON EDIT}} @alma1 along with editdata editors.
+		let dontRestore = (workingCopy.resource.path.includes('notebook-editor-') || workingCopy.resource.path.includes('dbo.'));
```

Comment:
I assume you're getting dbo because the editor title is the table name - e.g. `dbo.myTable`? That's not a usable solution though - the schema (dbo in this case) can be anything so hardcoding it like this is only going to work part of the time.

I'd suggest looking into where these working copies are being registered - is this behavior we can override when we create the editor?

Reply:
Its being called from untitledTextEditor, to avoid this, there needs to be an optional parameter for the class that checks if working copies are needed or not